src/main/java/io/in_toto/keys/RSAKey.java

```diff
@@ -137,7 +129,7 @@ public static RSAKey readPemBuffer(Reader reader)
         } catch (IOException e) {}
         return new RSAKey(kpr);
     }
-
+    
```

Comment:
and this trailing whitespace in this file.

Reply:
done. 1d38e7918d9a29523da6fa7cafe67b3aaf521c93